docs/sql-ref-literals.md

```diff
@@ -47,6 +47,10 @@ A string literal is used to specify a character string value.
     To represent unicode characters, use 16-bit or 32-bit unicode escape of the form `\uxxxx` or `\Uxxxxxxxx`,
     where xxxx and xxxxxxxx are 16-bit and 32-bit code points in hexadecimal respectively (e.g., `\u3042` for `あ` and `\U0001F44D` for `👍`).
 
+* **R**
```

Comment:
Hm .. I wonder if other DBMSes have their own syntax for this that we can refer to instead of the Python's ...

Reply:
Actually, I investigated raw string syntax support in other RDBMS beforehand, and I found there is no standard way to represent raw string literals.

In PostgreSQL, any special character isn't escaped by `\` unless a string literal starts with `E` prefix.
https://www.postgresql.org/docs/13/sql-syntax-lexical.html#SQL-SYNTAX-CONSTANTS

In MySQL, special characters in a string literal are not escaped if `NO_BACKSLASH_ESCAPES` is enabled.
https://dev.mysql.com/doc/refman/8.0/en/string-literals.html

In MsSQLServer, any special character isn't escaped by `\` but `STRING_ESCAPE` function can escape such characters.
https://docs.microsoft.com/en-us/sql/t-sql/functions/string-escape-transact-sql?view=sql-server-ver15

It's not a RDBMS but BigQuery supports raw string literals like Python.
https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#literals

So, I choose this syntax.

Reply:
I've updated the description to explain why `r'...'` and `r"..."` syntax is chosen.